prowler/providers/aws/services/rds/rds_instance_security_group_event_subscription/rds_instance_security_group_event_subscription.py

```diff
@@ -0,0 +1,34 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.rds.rds_client import rds_client
+
+
+class rds_instance_security_group_event_subscription(Check):
+    def execute(self):
+        findings = []
+        report = Check_Report_AWS(self.metadata())
+        report.status = "FAIL"
+        report.check_metadata.Severity = "medium"
+        report.status_extended = 'RDS security group event categories of "configuration change" and "failure" are not subscribed.'
+        report.region = rds_client.region
+        for db_event in rds_client.db_event_subscriptions:
```

Comment:
I think here we need to also check if there are RDS instances/clusters running in the region to improve results. Also I think this check is candidate for the `--scan-unused-services` flag, so:
- `--scan-unused-services` == True && no instances && not enabled events --> `FAIL`
- `--scan-unused-services` == False && no instances && not enabled events --> No finding

Reply:
Do you have an example of where this is used. I do not see this documented in the dev guide. I added a check to make sure that db_instances exist in that region.